src/startpage.css

```diff
@@ -19,28 +19,89 @@
 #startpage .button {
   padding-inline: 3em;
   padding-block: 1em;
-}
-
-.button.green {
-  color: #0d0;
+  filter: grayscale(0.25);
   transition:
-    ease-out color 250ms,
+    ease-out filter 250ms,
     ease-in-out scale 500ms;
 }
 
-.button.green:hover {
-  color: #0f0;
+#startpage .button:hover {
   scale: 1.1;
+  filter: grayscale(0);
+}
+
+.button.green {
+  color: #0f0;
```

Comment:
```suggestion
  color: green;
```

Reply:
green is actually `#080` and `#0f0` would be lime.
i think for readability better leave it as is, as we do it in the same format on red too.